### ISSUE TITLE
Initialize software bill of materials

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#13]
## User story
1. As a software developer
2. I want to initialize my software bill of materials found in the planning documents with the dependencies
3. So that I can document the dependencies of the project

## Acceptance criteria
* The sheet "Bill of Matrials" of the Planning documents is initialized with the project dependencies
* POs are informed via slack message so that they upload the homework on time

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
